lib/active_model/serializer/concerns/configuration.rb

```diff
@@ -10,6 +10,7 @@ module Configuration
         config = base.config
         config.collection_serializer = ActiveModel::Serializer::CollectionSerializer
         config.serializer_lookup_enabled = true
+        config.on_serializer_not_found = -> {}
```

Comment:
what if this is nil?
should it start out as nil?  does defaulting to an empty block feel weird?